export const createDataCloneError = () => {
    try {
        return new DOMException('', 'DataCloneError');
    }
    catch (err) {
        const exception = new Error();
        exception.code = 25;
        exception.name = 'DataCloneError';
        return exception;
    }
};
//# sourceMappingURL=/build/es2015/factories/data-clone-error.js.map